data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,381 @@ mission "Ringworld Debris: Quarg"
 		log "Asked the Quarg about the history of the Graveyard. Was told that the ringworld the Quarg had there was lost in their war with the Pug, and have learned about the previous inhabitants of the region, the Builders."
 		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called the "Ka'het," which stopped obeying their orders and rebelled against them.`
 		log "Factions" "Ka'het" `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then rebelled and led them to extinction. Even in the millennia that passed, they are still being produced in the Graveyard.`
+
+mission "Ka'het: Patir Reveal"
+	invisible
+	landing
+	to offer
+		has "Remnant: Cognizance 19: done"
+	on offer
+		event "patir appears" 3
+		fail
+
+event "patir appears"
+	system "Patir"
+		remove hidden
+
+mission "Ka'het: Patir Mystery 0"
+	invisible
+	landing
+	on enter "Patir"
+		dialog
+			`As the jumping sequence comes to an end, you find yourself orbiting a small black hole, surrounded by nothing but countless asteroids and debris. Even more interesting is the fact that the asteroid belt appears to be populated by small spaceships, similar in appearance to the Ka'het that infest the nearby systems but at the same time unlike any of the types you had seen before. Perhaps the Remnant can tell you more about this system and its sudden appearance.`
+		fail
+	to complete
+		never
+
+mission "Ka'het: Patir Mystery 1"
+	landing
+	name "Visit <waypoints>"
+	description "Bring Taely with you to visit the <waypoints> system before going to <destination>."
+	source
+		government "Remnant"
+	destination "Aventine"
+	waypoint "Patir"
+	passengers 1
+	cargo "Ka'het tracking equipment" 5
+	to offer
+		has "Ka'het: Patir Mystery 0: failed"
+		not "dusk busy"
```

Comment:
```suggestion
		not "Remnant: Dusk Busy"
```

This matches Cognizance, and will sort into the same area as other Remnant variables to make it easier to find.